src/backend/config/database.php

```diff
@@ -66,16 +66,38 @@
 
         // PHPunit testing main connection
         'testing' => [
-            'driver' => 'sqlite',
-            'database' => ':memory:',
+            // 'driver' => 'sqlite',
+            // 'database' => ':memory:',
+            // 'prefix' => '',
+            'driver' => 'mysql',
+            'host' => env('DB_HOST', '127.0.0.1'),
+            'port' => env('DB_PORT', '3306'),
+            'database' => env('DB_DATABASE', 'testing_test_company_db'),
+            'username' => env('DB_USERNAME', 'root'),
+            'password' => env('DB_PASSWORD', 'wF9zLp2qRxaS2e'),
+            'charset' => 'utf8mb4',
+            'collation' => 'utf8mb4_unicode_ci',
             'prefix' => '',
+            'strict' => true,
+            'engine' => null,
         ],
```

Comment:
I don't like that the tests use their own **connection** (own database yes, but not own connection). We actually have the connection names `micro_power_manager` and `shard` hardcoded in a lot of places in the code base. So, I'm doubtful if using different connection names will even work.

Do you think it's possible to remove the two testing connections and the envs `DB_CONNECTION` and `TEST_DB_CONNECTION`? 

We could instead use a different `DB_DATABASE` env when running test.

However, I'm not sure how many side effects this will have. So, I'd say we move this to a different PR

Reply:
I will have to give it a try. I will make things cleaner if we have the same connections